packages/basebridge/src/BaseBridge.test.ts

```diff
@@ -0,0 +1,117 @@
+import { bridge } from './BaseBridge'
+import {
+  BASE_ERC20_ABI,
+  BASE_ERC20_BRIDGE_ADDRESS,
+  BASE_ETH_ABI,
+  BASE_ETH_BRIDGE_ADDRESS,
+  ETH,
+  ETHEREUM_ERC20_ABI,
+  ETHEREUM_ERC20_BRIDGE_ADDRESS,
+  ETHEREUM_ETH_ABI,
+  ETHEREUM_ETH_BRIDGE_ADDRESS,
+} from './constants'
+import { failingTestCases, passingTestCases } from './test-transactions'
+import { Chains } from '@rabbitholegg/questdk-plugin-utils'
+import { apply } from '@rabbitholegg/questdk/filter'
+import { parseEther } from 'viem'
+import { describe, expect, test } from 'vitest'
+
+describe('Given the basebridge plugin', () => {
+  describe('When handling the bridge action', () => {
+    describe('should return a valid action filter', () => {
+      test('when bridge eth from ethereum to base', async () => {
```

Comment:
Personally I find these tests to be very cumbersome and not very beneficial. Lately I have been replacing these action filter tests for this set of generic tests. Up to you if you want to replace it with this or update the existing ones to work with the proposed changes.  
      
      
     ``` 
      test('when making a valid bridge action', async () => {
        const filter = await bridge({
          sourceChainId: 1,
          destinationChainId: 56,
        })
        expect(filter).toBeTypeOf('object')
        expect(Number(filter.chainId)).toBe(1)
        if (typeof filter.to === 'string') {
          expect(filter.to).toMatch(/^0x[a-fA-F0-9]{40}$/)
        } else {
          // if to is an object, it should have a logical operator as the only key
          expect(filter.to).toBeTypeOf('object')
          expect(Object.keys(filter.to)).toHaveLength(1)
          expect(
            ['$or', '$and'].some((prop) =>
              Object.hasOwnProperty.call(filter.to, prop),
            ),
          ).to.be.true
          expect(Object.values(filter.to)[0]).to.satisfy((arr: string[]) =>
            arr.every((val) => val.match(/^0x[a-fA-F0-9]{40}$/)),
          )
        }
        // Check the input property is the correct type and has a valid filter operator
        expect(filter.input).toBeTypeOf('object')
        expect(
          ['$abi', '$abiParams', '$abiAbstract', '$or', '$and'].some((prop) =>
            Object.hasOwnProperty.call(filter.input, prop),
          ),
        ).to.be.true
      })
      ```